CHANGELOG.md

```diff
@@ -20,46 +20,43 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ### Highlights
 
-- Allow including draft orders in voucher usage. New `includeDraftOrderInVoucherUsage` flag allows defining if vouchers used in draft orders should be counted into voucher usage. - #14288 by @zedzior, @IKarbowiak, @michal-macioszczyk
-
+- Allow including draft orders in voucher usage. The new `includeDraftOrderInVoucherUsage` flag allows defining if vouchers used in draft orders should be counted into voucher usage. - #14288 by @zedzior, @IKarbowiak, @michal-macioszczyk
   - Add `includeDraftOrderInVoucherUsage` to `OrderSettings`
   - Add `includeDraftOrderInVoucherUsage` to `OrderSettingsInput`
 
 ### Breaking changes
 
-- Optimize number of queries in bulk mutations when calling webhooks. This change affects only users of open-source Saleor, who have their own custom plugin implementations. To adjust to this change, the `webhooks` parameter should be added to any of the affected method. Affected methods:
+- Optimize the number of queries in bulk mutations when calling webhooks. This change affects only users of open-source Saleor, who have their custom plugin implementations. To adjust to this change, the `webhooks` parameter should be added to any of the affected methods. Affected methods:
   - `attribute_updated`
   - `attribute_deleted`
   - `attribute_value_deleted`
   - `promotion_deleted`
   - `staff_deleted`
-- Saleor will no longer reattempt delivery for webhooks that return non-transient HTTP errors (400, 404 etc.) or redirects - #14566 by @patrys
-- **Feature preview breaking change**:
+- Saleor will no longer reattempt delivery for webhooks that return non-transient HTTP errors (400, 404, etc.) or redirects - #14566 by @patrys
+- Feature preview breaking changes:
   - Drop `defaultTransactionFlowStrategy` from `OrderSettings` type. Use `PaymentSettings.defaultTransactionFlowStrategy` instead. Drop `defaultTransactionFlowStrategy` from `OrderSettingsInput` type. Use `PaymentSettingsInput.defaultTransactionFlowStrategy` instead. - #14671 by @korycins
 
 ### GraphQL API
 
-- Fix draft order voucher assignment - #14336 by @IKarbowiak
 - Allow add multiple codes per voucher - #14123 by @SzymJ, @IKarbowiak, @michal-macioszczyk, @zedzior
   - Add `VoucherInput.addCodes` to `voucherCreate` and `voucherUpdate` mutations.
-  - Add `Voucher.singleUse` flag.
+  - Add the `Voucher.singleUse` flag.
   - Deprecate `OrderBulkCreateInput.voucher`.
   - Deprecate `VoucherInput.code` in `voucherCreate` and `voucherUpdate` mutations.
   - Add `exportVoucherCodes` mutation.
   - Add `voucherCodeBulkDelete` mutation.
```

Comment:
flag got the, but two mutations below didn't

Reply:
👮 Yeah, Grammarly did that but I think you're right.